File: Javascript/TallerJS/Ejercicio5.js
// 6.Pedir dos números y decir si son múltiplos o no.


let num1 = parseInt(prompt("Ingrese un numero"))
let num2 = parseInt(prompt("Ingrese otro numero"))

let resultado = num2 % num1

if(num1 < 0 || num2 < 0){
    alert("Algun numero no es valido")
}else{
    if(resultado == 0){
        console.log("El numero ", num1, "Es multiplo de ", num2)
    }else{
        console.log("El numero ", num1, "No es multiplo de ", num2)
    }
}


// console.log(resultado, num1, num2)

File: Modulo1/Python/Condicionales/Ejercicio5.py
# 6.Pedir dos números y decir si son múltiplos o no.


num1 = int(input("Ingrese un numero "))
num2 = int(input("Ingrese otro numero "))

resultado = num2 % num1

if num1 < 0 or num2 < 0 :
    print("Algun numero no es valido")
else:
    if resultado == 0 :
        print("El numero ", num1, "Es multiplo de ", num2)
    else:
        print("El numero ", num1, "No es multiplo de ", num2)
    



#  console.log(resultado, num1, num2)